common/src/main/java/com/example/android/uamp/media/PackageValidator.kt

```diff
@@ -116,6 +116,8 @@ class PackageValidator(context: Context, @XmlRes xmlResId: Int) {
             callingUid == Process.SYSTEM_UID -> true
             // If the app was signed by the same certificate as the platform itself, also allow it.
             callerSignature == platformSignature -> true
+            // If the app package name starts with com.example, such as the MCT
+            callingPackage.startsWith("com.example") -> true
```

Comment:
We probably don't want to encourage this level of access. I know @nic0lette had some more thoughts on this.

Reply:
Yeah, I would rather not do this. Developers can add their own package/signature to the allow list, or they can add this sort of code specifically if they'd like, but, given this is a set of rules we'd like app developers to actually adopt, I don't want to have this be so open.

Reply:
Ah, makes sense. Removed!